value.go

```diff
@@ -293,7 +293,7 @@ func (vlog *valueLog) iterate(lf *logFile, offset uint32, fn logEntry) (uint32,
 		}
 
 		var vp valuePointer
-		vp.Len = uint32(headerBufSize + len(e.Key) + len(e.Value) + 4) // len(crcBuf)
+		vp.Len = uint32(headerBufSize + len(e.Key) + len(e.Value) + crc32.Size)
```

Comment:
This len can be kept. So, if someone switches how crcBuf is created, then we this would stay in sync with automatically.

Reply:
The value is *currently* hard-coded (`len(crcBuf)` is just a comment). Is that what you're referring to?

Reply:
Ah.. Should be len(crcBuf) then.

Reply:
I'll take care of it later.